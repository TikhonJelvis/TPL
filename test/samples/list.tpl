print []
print [1,2,3]
print [null, 1, 2, true, false]
print [[], [], []]
print [[[[[[[[[[1,2,3],4],5],6],7],8],9],10],11],12]