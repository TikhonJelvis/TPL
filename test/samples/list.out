[]
[1, 2, 3]
[null, 1, 2, true, false]
[[], [], []]
[[[[[[[[[[1, 2, 3], 4], 5], 6], 7], 8], 9], 10], 11], 12]
null
